pandas/io/parquet.py

```diff
@@ -512,17 +511,24 @@ def read_parquet(
     DataFrame
     """
     impl = get_engine(engine)
-
-    use_nullable_dtypes = (
-        use_nullable_dtypes
-        if use_nullable_dtypes is not lib.no_default
-        else using_nullable_dtypes()
-    )
+    if use_nullable_dtypes is not lib.no_default:
+        msg = (
+            "The argument 'use_nullable_dtypes' is deprecated and will be removed "
+            "in a future version."
+        )
+        if use_nullable_dtypes is True:
+            msg += (
+                "Use dtype_backend='numpy_nullable' instead of use_nullable_dtype=True."
+            )
+        warnings.warn(msg, FutureWarning, stacklevel=find_stack_level())
```

Comment:
What would people think about not yet actually deprecating this keyword? (for parquet (and feather?), i.e. where it already existed) 
That gives us a bit more time to see if what we decide now with dtype_backed is actually the way to go.

Reply:
Hmm we could still un-deprecate this later in an 2.x release. I think I would rather prefer being consistent now with `dtype_backend` rather than allowing both `use_nullable_dtypes` and `dtype_backend`

Reply:
Yeah agreed I'd rather make this consistent as well